src/OrleansLoggerProvider/OrleansLogger.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+
+namespace Orleans.Extensions.Logging
+{
+    /// <summary>
+    /// OreansLogger supports legacy orleans logging features, including <see cref="ILogConsumer"/>, <see cref="ICloseableLogConsumer">,
+    /// <see cref="IFlushableLogConsumer">, <see cref="Severity">. 
+    /// </summary>
+    [Obsolete]
```

Comment:
All obsolete attributes should have a description of what should be the replacement. You should probably define a constant with this string, as you will use it in multiple places.